store/tikv/2pc.go

```diff
@@ -224,9 +224,19 @@ func (c *twoPhaseCommitter) doActionOnKeys(bo *Backoffer, action twoPhaseCommitA
 	if action == actionCommit {
 		// Commit secondary batches in background goroutine to reduce latency.
 		twoPhaseCommitGP.Go(func() {
+			start := time.Now()
+			secondaryLockCleanupWorkerGauge.Inc()
+			defer func() {
+				secondaryLockCleanupWorkerGauge.Dec()
+				secondaryLockCleanupHistogram.WithLabelValues("secondary_lock_cleanup").Observe(time.Since(start).Seconds())
```

Comment:
I think the tag name is duplicated with the metric name here.

Reply:
seem you don't need to use a MetricVec here, only a Metric is enough.

Reply:
I think I'd better to make a distinction between commit and rollback here.